src/main/java/mandel/fallingsand/Main.java

```diff
@@ -0,0 +1,28 @@
+package mandel.fallingsand;
+
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Sand sand = new Sand(50, 10);
+        sand.randomSand(50);
+        Scanner input = new Scanner(System.in);
+        try {
+            sand = new Sand(Integer.parseInt(args[0]), Integer.parseInt(args[1]));
+            sand.randomSand(Integer.parseInt(args[2]));
```

Comment:
You do this on line 8-9.
Why are you doing it again?

Reply:
I thought we were supposed to use command line, but it wouldn't work if I just did it inside the try. Should I get rid of the whole try-catch and leave it on lines 8-9?

Reply:
Yes.

Reply:
I deleted the extra code, but now my build is failing again